### DISCUSSION TITLE
macOS Capacity shows as 0 mAh

### DISCUSSION THREAD

**Discussion by sbarissonmez** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1251] [Q&A]
I am using the AIDente Pro 1.27 version on MacBook Pro 14" M3 Sonoma 14.5 and macOS battery health capacity shows as 0mAh

[
<img width="544" alt="SCR-20240624-cwwr" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/13288840/7fa72b28-2752-4045-b779-c8a7a55580b4">
](url)

**Comment by MatthiasKerbl** [comment]
Hi @sbarissonmez,

Thank you for your feedback and we are sorry about this inconvenience.
Do you mind sharing a debug file so that we can assess this problem in more detail?
You can find a guide on [How to generate and share a debug file](https://apphousekitchen.com/how-to-generate-and-share-a-debug-file/) on our blog.

Best,
Matthias

**Comment by sbarissonmez** [comment]
Hi @MatthiasKerbl,

Thank you for your quick response. You can download the debug file following link;

[aldentepro-debug-2024-06-24-16:23:11.pasta.zip](https://github.com/user-attachments/files/15956329/aldentepro-debug-2024-06-24-16.23.11.pasta.zip)

**Comment by sbarissonmez** [comment]
Hi @MatthiasKerbl 

Seems like removed on the 1.27.2 version of the macOS Capacity data. Also, I can't see the new version power flow when open the application window but appears on the popover window.

<img width="920" alt="SCR-20240708-mrli" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/13288840/371457de-aa02-45ac-a9cb-e0b23ec4cef1">

**Reply by MatthiasKerbl** [reply]
Hi @sbarissonmez,

We have removed the macOS Capacity until we get our hands on a 14" M3 MacBook Pro. Also, the Power Flow graph will be added again in the Stats tab with a small update soon.

Best,
Matthias
